config/application.rb

```diff
@@ -35,7 +34,7 @@ class Application < Rails::Application
 
     # Locale configuration
     config.i18n.load_path += Dir[Rails.root.join('config', 'locales', '**', '*.{rb,yml}')]
-    config.i18n.available_locales = %w[en mr-IN pa-Guru-IN bn-BD en_BD en_IN bn-IN hi-IN ta-IN te-IN or-ET ti-ET]
+    config.i18n.available_locales = %w[en mr-IN pa-Guru-IN bn-BD en_BD en_IN en-IND bn-IN hi-IN ta-IN te-IN or-ET ti-ET]
```

Comment:
If we're afforded flexibility, can we name the new locale `en-SEED`?

Reply:
Can't, this unforuntately is the locale used by Fake for Indian things. It uses `en-IND` instead of `en-IN` for some reason. This allows us to extend Faker's corpus.